instance/tasks.py

```diff
@@ -339,3 +345,103 @@ def cleanup_old_betatest_users():
         users_to_delete = queryset.filter(is_active=False, profile__modified__lte=delete_cutoff)
         logger.info('Deleting %s users.', users_to_delete.count())
         users_to_delete.delete()
+
+
+class KillZombiesRunner:
+    """
+    Helper class to run `kill_zombies_periodically`
+    """
+
+    def __init__(self):
+        self.enabled: bool = bool(os.environ.get("KILL_ZOMBIES_ENABLED", False))
+        self.region: str = os.environ.get("KILL_ZOMBIES_REGION", "gra")
+        self.recipient: str = os.environ.get(
+            "KILL_ZOMBIES_WARNING_RECIPIENT", "ops@opencraft.com"
```

Comment:
@spokerman12, I would recommend against using an actual email address specific to OpenCraft in a public repository like this.

Reply:
Thank you for suggesting this. Added